### PR TITLE
Clinic database schema

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this Milestone, we did the following activities:

- Diagram a data model clearly and explain what it's doing.
- Create a file named `schema_based_on_diagram.sql`
- Join tables from many-to-many relationships
- Add FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+CREATE TABLE invoices(
+    id INT NOT NULL GENERATED BY DEFAULT AS IDENTITY,
+    total_amount DECIMAL(8, 2) NOT NULL,
+    generated_at TIMESTAMP NOT NULL,
+    payed_at TIMESTAMP NOT NULL,
+    medical_history_id INT NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_medical_histories FOREIGN KEY(Id) REFERENCES medical_histories(id)
+);
```
Comment (Ghiftee) [inline schema_based_on_diagram.sql]: - [ ] You currently have two tables named ```invoices``` and with similar content. Kindly adjust this

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @cvalencia1991 and @Kelvin-Ben 👋🏽  

Good job working on the project so far! 
✔️ You created tables
✔️ You created foreign keys
However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (cvalencia1991) [reply schema_based_on_diagram.sql]: ok im going to delete that thank you @Ghiftee

**Review by codepantha — APPROVED** [review]
# STATUS; APPROVED 🟢 ✔️ 

## Hi team 👋, 

You have done a great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cvalencia1991** [comment]
thank you 😄

**Pull request merged**